cogs/fixtures.py

```diff
@@ -91,7 +91,17 @@ async def next(self, ctx):
     async def results(self, ctx):
         fixtures = parseResults()
         body = findResults(fixtures)
-        await ctx.send(f"```{body}```")
+
+        embed = discord.Embed(
+            color=0x9C824A,
+            description=f"```{body}```")
+
+        embed.set_author(
+            name="Last 3  results",
+            icon_url="https://resources.premierleague.com/premierleague/badges/t3.png"
```

Comment:
We're already defining club badges icon location in another file, would it be worth consolidating it to 1 place in case that link changes at some point

Reply:
+1